common/app/common/commercial/ContainerModel.scala

```diff
@@ -23,7 +30,7 @@ case class ContainerContent(
 
 object ContainerModel {
 
-  def fromPressedCollection(collection: PressedCollection): ContainerModel = {
+  def fromPressedCollection(edition: Edition)(collection: PressedCollection): ContainerModel = {
```

Comment:
Is there any reason to have two parameter lists here rather than one? (this is more me being curious than suggesting we merge them)